src/events/janus_rabbitmqevh.c

```diff
@@ -346,6 +352,26 @@ int janus_rabbitmqevh_init(const char *config_path) {
 }
 
 int janus_rabbitmqevh_connect(void) {
+	uint8_t retry_interval = 5; // Retry interval in seconds
+
+	while (!g_atomic_int_get(&stopping)) {
+		if (janus_rabbitmqevh_tryconnect() == 0) {
```

Comment:
Code style: we prefer no space between the check (`while`, `if` etc.) and what comes next. As such, this should be

    while(...

and

    if(...

There's a few other occurrences of the same, so please change them there too.